source/overloading.tex

```diff
@@ -1750,6 +1785,21 @@
 };
 
 E e1 = {1, 2};                  // OK, deduces \tcode{E<int>}
+
+template <typename... T>
+struct Types {};
+
+template <typename... T>
+struct F : Types<T...>, T... {};
+
+struct X {};
+struct Y {};
+struct Z {};
+struct W { operator Y(); };
+
+F f1 = {Types<X, Y, Z>{}, {}, {}};      // OK, F<X, Y, Z> deduced
+F f2 = {Types<X, Y, Z>{}, X{}, Y{}};    // OK, F<X, Y, Z> deduced
+F f3 = {Types<X, Y, Z>{}, X{}, W{}};    // error: conflicting types deduced; operator Y not considered
```

Comment:
use \tcode for code expressions in comments

Reply:
Fixed.